modules/features/discover/src/main/java/au/com/shiftyjelly/pocketcasts/discover/view/DiscoverFragment.kt

```diff
@@ -214,6 +220,20 @@ class DiscoverFragment : BaseFragment(), DiscoverAdapter.Listener, RegionSelectF
             FirebaseAnalyticsTracker.navigatedToDiscover()
         }
     }
+    private fun sortContent(content: List<Any>): MutableList<Any> {
+        val mutableContentList = content.toMutableList()
+
+        if (FeatureFlag.isEnabled(Feature.CATEGORIES_REDESIGN)) {
+            val categoriesIndex = mutableContentList.indexOfFirst { it is DiscoverRow && it.type is ListType.Categories }
+
+            if (categoriesIndex != -1) {
+                val categoriesItem = mutableContentList.removeAt(categoriesIndex)
+                mutableContentList.add(0, categoriesItem)
+            }
+        }
+
+        return mutableContentList
+    }
```

Comment:
I will sync with iOS and backend if there is a way to remotely set categories at the top without needing this so I can address it to another PR